format/testfiles/test.rego.formatted

```diff
@@ -19,7 +19,10 @@ foo[x] {
 	g(x, "foo") = z
 }
 
-globals = {"fizz": "buzz", "foo": "bar"}
+globals = {
+	"foo": "bar",
+	"fizz": "buzz",
+}
```

Comment:
This reverts the crucial bit of #3823 that _should_ have given this bug away but that I didn't notice in its full extent.